src/main/scala/firrtl/Visitor.scala

```diff
@@ -322,6 +322,10 @@ class Visitor(infoMode: InfoMode) extends AbstractParseTreeVisitor[FirrtlNode] w
         case "smem" =>
           val (tpe, size) = visitCMemType(ctx.`type`())
           CDefMemory(info, ctx.id(0).getText, tpe, size, seq = true, readUnderWrite = visitRuw(Option(ctx.ruw)))
+        case "memaccess" =>
+          DefMemAccess(info, ctx.id(0).getText, visitExp(ctx_exp(0)), visitExp(ctx_exp(1)), visitExp(ctx_exp(2)))
+        case "maskedwrite" =>
+          MemMaskedWrite(info, visitExp(ctx_exp(0)), visitExp(ctx_exp(1)), visitExp(ctx_exp(2)), visitExp(ctx_exp(3)))
```

Comment:
you added the keyword `memwrite` to the grammar, why are you matching on `maskedwrite` here?